rules/CodingStyle/ValueObject/NodeToRemoveAndConcatItem.php

```diff
@@ -10,18 +10,18 @@
 final class NodeToRemoveAndConcatItem
 {
     public function __construct(
-        private Expr $removedExpr,
-        private Node $concatItemNode
+        private Expr $expr,
+        private Node $node
```

Comment:
This should be preserved as it contains name of external domain.
If `Type = $<some>Type`, the content name should be kept.

Reply:
Do you mean we need to keep variable for type instance of `Node`?

Reply:
Both of them

Reply:
fixed https://github.com/rectorphp/rector-src/pull/168/commits/bb611aba879334e187d5273917be936321749102